tools/lib/provision.py

```diff
@@ -142,6 +142,13 @@
     "libappindicator1",
     "xdg-utils",
     # Puppeteer dependencies end here.
+
+    # Fixes missing sasl.h headers during
+    # python-ldap installation.
+    "libsasl2-dev",
+    "gcc",
+    "python-dev",
```

Comment:
We already have `gcc` via `build-essential`, and `python-dev` is Python 2. What do we need those for?

Reply:
Actually, when I faced this issue, I installed the three of them together. Ideally, we should not need `python-dev`, but I am not sure.